plugins/Dashboard/Model.php

```diff
@@ -133,10 +133,11 @@ public function updateLayoutForUser($login, $idDashboard, $layout)
     public static function install()
     {
         $dashboard = "login VARCHAR( 100 ) NOT NULL ,
-					  iddashboard INT NOT NULL ,
-					  name VARCHAR( 100 ) NULL DEFAULT NULL ,
-					  layout TEXT NOT NULL,
-					  PRIMARY KEY ( login , iddashboard )";
+                      iddashboard INT NOT NULL ,
+                      name VARCHAR( 100 ) NULL DEFAULT NULL ,
+                      layout TEXT NOT NULL,
+                      idsite bigint(20) NOT NULL ,
```

Comment:
`idsite` is defined as `int(11)` in most other tables, so would be enough to use that.